qutebrowser/browser/history.py

```diff
@@ -156,9 +157,18 @@ def __init__(self, progress, parent=None):
         self.completion = CompletionHistory(parent=self)
         self.metainfo = CompletionMetaInfo(parent=self)
 
+        # Get a string of all patterns
+        patterns = "\n".join([str(pattern) for pattern in
+                              config.cache['completion.web_history.exclude']])
```

Comment:
While this should probably work, you can get a string out of the config system directly via:

```suggestion
        config.instance.get_str('completion.web_history.exclude')
```

Reply:
Change applied in commit 93ecd8f